common/rtc_manager_rviz_plugin/src/rtc_manager_panel.cpp

```diff
@@ -89,7 +92,7 @@ bool isPathChangeModule(const uint8_t module_type)
     module_type == Module::EXT_REQUEST_LANE_CHANGE_LEFT ||
     module_type == Module::EXT_REQUEST_LANE_CHANGE_RIGHT || module_type == Module::AVOIDANCE_LEFT ||
     module_type == Module::AVOIDANCE_RIGHT || module_type == Module::PULL_OVER ||
-    module_type == Module::PULL_OUT) {
+    module_type == Module::PULL_OUT || module_type == Module::INVALID_LANELET) {
```

Comment:
This module will modify the output path shape? If it will modify only path velocity, this line diff is unnecessary.